packages/grant-explorer/src/features/round/ViewCartPage/SummaryContainer.tsx

```diff
@@ -275,13 +280,78 @@ export function SummaryContainer() {
     }
   }, [totalDonationAcrossChainsInUSDData]);
 
+  /* Matching estimates are calculated per-round */
+  const matchingEstimateParamsPerRound =
+    rounds?.map((round) => {
+      const projs = projects.filter((project) => project.roundId === round.id);
```

Comment:
Since only the first project is being used (if I'm reading correctly), this could be replaced with a `find`.

Reply:
True and fixed.